packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js

```diff
@@ -79,6 +79,7 @@ describe('TiktokAds Track event', () => {
               product_id: 'abc',
               category: 'Merch',
               name: 'Food',
+              brand: '',
```

Comment:
why will this be present?

Reply:
Its based upon rudderstack ecom spec in general and it is for input body not output